frontend/src/pages/modelServing/__tests__/utils.spec.ts

```diff
@@ -226,3 +227,33 @@ describe('getTemplateEnabledForPlatform', () => {
     ).toBeTruthy();
   });
 });
+
+describe('getEnabledPlatformsFromTemplate', () => {
+  it('should return only MULTI if no annotation', () => {
+    const teamplateAllPlatforms = mockServingRuntimeTemplateK8sResource({
+      platforms: [],
+    });
+    expect(getEnabledPlatformsFromTemplate(teamplateAllPlatforms)).toEqual([
+      ServingRuntimePlatform.MULTI,
+    ]);
+  });
```

Comment:
Could you add a test case for when the `opendatahub.io/modelServingSupport` annotation is missing. Copy this test case and delete the annotation after creating the mock serving runtime template.

Reply:
oh right, forgot to add that, thank s a lot @christianvogt